std/traits.d

```diff
@@ -5154,6 +5157,22 @@ unittest
 }
 
 /**
+Detect whether `T` is a scalar type that promotes to an integral type (that is, a built-in
+integral, character, or boolean type).
+ */
+enum bool isFixedPoint(T) = isIntegral!T || isSomeChar!T || isBoolean!T;
```

Comment:
Interesting rationale, but the converse of floating point is not fixed point; the latter is a mere alternative for implementing approximations of real numbers. Integrals and their ilk are a distinct matter.

How frequent is this likely to be? Should it include integral `enum` types?